h/sql_tasks/tasks/report/create_from_scratch/01_authorities/02_initial_fill.sql

```diff
@@ -0,0 +1,45 @@
+-- You can get values to maintain this table with the following query:
+
+-- SELECT
+--     -- Create a fake "primary" key
+--     created::date,
+--     authority
+-- FROM (
+--     SELECT
+--         authority,
+--         MIN(created) as created
+--     FROM "group"
+--     GROUP BY authority
+--     ORDER BY created
+-- ) as data
+
+-- These values are required to be globally unique
+
+TRUNCATE report.authorities;
+
+INSERT INTO report.authorities (id, created, authority)
+VALUES
+    -- Values we care about the most
+    (1, '2015-09-22', 'hypothes.is'),
+    (2, '2018-10-12', 'lms.hypothes.is'),
+    (3, '2022-01-21', 'lms.ca.hypothes.is'),
+
+    -- Others
+    (1001, '2017-04-28', 'test.elifesciences.org'),
```

Comment:
Yep, hopefully. Sort of mixed evidence for that here. We can see there aren't many for a start, which is good, but we can also see that some were added recently.

I suspect this will be a squeaky wheel scenario, where if people want reports for something they will ask, and that will jog us to remember.